docs/how-to-guides/upgrade-etcd.md

```diff
@@ -0,0 +1,112 @@
+# Upgrading etcd
+
+## Contents
+
+- [Introduction](#introduction)
+- [Steps](#steps)
+  - [Step 1: Find out the IP and SSH](#step-1-find-out-the-ip-and-ssh)
+  - [Step 2: Create necessary directories with correct permissions](#step-2-create-necessary-directories-with-correct-permissions)
+  - [Step 3: Upgrade etcd](#step-3-upgrade-etcd)
+  - [Step 4: Verify upgrade](#step-4-verify-upgrade)
+  - [Step 5: Verify using `etcdctl`](#step-5-verify-using-etcdctl)
+
+## Introduction
+
+[Etcd](https://etcd.io/) is the most crucial component of a Kubernetes cluster. It stores the cluster state.
+
+This document will provide step by step guide on upgrading etcd in Lokomotive.
+
+## Steps
+
+Repeat the following steps on all the controller node one node at a time.
+
+### Step 1: Find out the IP and SSH
+
+Find the IP of the controller node by visiting the cloud provider dashboard and ssh into it.
+
+```bash
+ssh core@<IP Address>
+```
+
+### Step 2: Create necessary directories with correct permissions
+
+Latest etcd (`>= v3.4.10`) necessitates the data directory permissions to be `0700`, accordingly change the permissions. Verify the permissions are changed to `rwx------`.
+
+> **NOTE**: This step is needed only for the Lokomotive deployment done using `lokoctl` version `< 0.4.0`.
+
+```bash
+sudo chmod 0700 /var/lib/etcd/
+sudo ls -ld /var/lib/etcd/
+```
+
+If the node reboots, we need the right settings in place so that `systemd-tmpfile` service does not alter the permissions of the data directory. To make the changes made above persistent run the following command:
```

Comment:
made above seems redundant.

```suggestion
If the node reboots, we need the right settings in place so that `systemd-tmpfile` service does not alter the permissions of the data directory. To make the changes persistent run the following command:
```